servicemodules/discord.py

```diff
@@ -243,44 +243,7 @@ async def on_socket_raw_receive(_) -> None:
                 for comment in new_comments:
                     if "?" in comment["text"]:
                         self.utils.add_youtube_question(comment)
-            # add_question should maybe just take in the dict, but to make sure
-            # nothing is broken extra fields have been added as optional params
-            # This is just checking if there _are_ questions
-            question_count = self.utils.get_question_count()
```

Comment:
this was using the old wiki and I didn't find open issue to migrate the code to Coda so assuming someone did it in some other module and we can delete

if not the case, feel free to add to todo list @tayler6000 (ideally with description from @robertskmiles and not just trying to guess what this old code was trying to do before people forgot to update it and it was rotting trying to do something about data from wiki)

Reply:
this is now in the `questions.py` module.